cognitive_mapping_and_planning/tfcode/vision_baseline_lstm.py

```diff
@@ -196,6 +197,7 @@ def combine_setup(name, combine_type, embed_img, embed_goal, num_img_neuorons=No
       out = embed_goal
     else:
       logging.fatal('Undefined combine_type: %s', combine_type)
+      raise ValueError('Undefined combine_type: %s', combine_type)
```

Comment:
@martinwicke thanks for looking into this. I am not sure what you mean: does `logging.fatal` raise? I thought it didn't and that's why we needed to